components/listings/ListingDetails.tsx

```
@@ -139,11 +142,304 @@ export const ListingDetails: React.FC<ListingDetailsProps> = ({ listing }) => {
+                    <address className="not-italic text-gray-700 dark:text-gray-200">
+                      {isValidString(listing.address.addressLine1) && (
+                        <>{listing.address.addressLine1}<br /></>
+                      )}
+                      {isValidString(listing.address.addressLine2) && (
+                        <>{listing.address.addressLine2}<br /></>
+                      )}
+                      {isValidString(listing.address.addressLine3) && (
+                        <>{listing.address.addressLine3}<br /></>
+                      )}
+                      {isValidString(listing.address.postcode) && (
+                        <>{listing.address.postcode}</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)